pkg/api/types.go

```diff
@@ -569,6 +569,7 @@ type MasterProfile struct {
 	AvailabilityZones         []string          `json:"availabilityZones,omitempty"`
 	SinglePlacementGroup      *bool             `json:"singlePlacementGroup,omitempty"`
 	AuditDEnabled             *bool             `json:"auditDEnabled,omitempty"`
+	UltraSSDEnabled           *bool             `json:"ultraSSDEnabled,omitempty"`
```

Comment:
@jackfrancis looks like I need to remove this `UltraSSDEnabled` from MasterProfile since master node does not need `UltraSSDEnabled`, and I have already added `UltraSSDEnabled` for AgentPoolProfile.

Reply:
We don't think there's any story where UltraSSDEnabled would be useful for a master vm?

Reply:
correct, UltraSSDEnabled is only for user's workload on agent node, let me remove that property from master node first.

Reply:
To be clear, ultraSSDEnabled is only needed for agent nodes according to the requirements you're developing against, right? There's no reason why it wouldn't work on a master vm, correct?

@ritazh @craiglpeters FYI, it sounds like we'll introduce this functionality on agent nodes first of all. But assuming there's nothing that forbids it from being useful on a master node, we want to be aware that that chunk of work will be in addition (and after) this chunk of work from @andyzhangx.

Reply:
@jackfrancis finally I also added this flag to master node, it's only a small fix:
https://github.com/Azure/aks-engine/pull/2905/commits/f4612ab96d11c93aff2c7eb9de49b4b0ed868f07